modules/hotmaps/1.0/config/default.yaml

```diff
@@ -0,0 +1,92 @@
+lcr-modules:
+    
+    hotmaps:
+
+        inputs:
+            # Available wildcards: {seq_type}
+            # No genome_build wildcard because only grch37 is compatible!
+            # Make sure input MAF paths are for grch37 only
+            input_maf: "__UPDATE__"
+            subsetting_categories: "__UPDATE__"
+
+        maf_processing:
+            sample_sets: ["__UPDATE__"]
+            prepare_mafs: "{SCRIPTSDIR}/generate_smg_inputs/1.0/generate_smg_inputs.R"
+            include_non_coding: False
+            blacklists: []
+            blacklist_drop_threshold: 4
+        
+        vcf2maf:
+            # Path pointing to VEP files for running vcf2maf e.g "ref/ensembl_vep_cache"
+            vep_cache: "__UPDATE__"
+
+        # Path to HotMAPS repo containing updated scripts
+        hotmaps_repo: "https://github.com/morinlab/HotMAPS/archive/refs/heads/master.zip"
+
+        pdb_stucture_dirs:
+            # modbase_dir: set to the directory containing the ModBase theoretical protein structures 
```

Comment:
Can you please here maybe add comments suggesting how to get these modbase and pdb?

Reply:
Added more info!